tasks/go.py

```diff
@@ -41,7 +41,7 @@ def fmt(ctx, targets, fail_on_fmt=False):
     Example invokation:
         inv fmt --targets=./pkg/collector/check,./pkg/aggregator
     """
-    if isinstance(targets, basestring):
+    if isinstance(targets, str):
```

Comment:
These validations are not backward compatible, a unicode string on `py2` would pass with `basestring` and would fail with the new code. 

To be honest, I'm not sure if this is a problem. It might be acceptable to just leave python2 in the past and focus on making this python3 compliant.